### DISCUSSION TITLE
Dialogue waits until a tween called from a singleton finishes to continue the dialogue. How can i make it "ignore" the tween?

### DISCUSSION THREAD

**Discussion by kevinsavala** [nathanhoad/godot_dialogue_manager#452] [Q&A]
Like making the tween run in the background. 
`
Dialogue...

do StartTween(2)

Next dialogue...
`
When tween starts, it runs for the seconds passed as parameter, but the text panel hides for the duration of the tween. I want it to start the tween and still keep the dialogue going. Thanks.

**Comment by DanielSparta** [comment]
Please provide more information about what your trying to do and I will try to help you

**Comment by nathanhoad** [comment]
What does the implementation for `StartTween` look like?
